src/lib/flow/codegen/blocklayer/block2ast.ts

```
@@ -635,119 +647,296 @@ export class CodeGenASTGenerator {
+				case TypedInputEnum.Raw:
+					task.setResult({
+						type: LiteralEnum.Raw,
+						value: source.value
+					});
+					continue;
+				case TypedInputEnum.Table:
+					const tableResult: LiteralValue = {
+						type: LiteralEnum.Table,
+						value: new Array(source.value.length), // Pre-allocate array
+						inline: source.inline
+					};
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)